cmd/commands/service/command.go

```diff
@@ -111,9 +113,8 @@ type serviceCommand struct {
 // Run runs a command
 func (sc *serviceCommand) Run(ctx *cli.Context) (err error) {
 	arg := ctx.Args().Get(0)
-	// If no service type specified we are starting wireguard only.
-	// Other services could be started only explicitly.
-	serviceTypes := []string{wireguard.ServiceType}
+	// If no service type specified we are starting wireguard, scraping and data_transfer.
+	serviceTypes := []string{wireguard.ServiceType, scraping.ServiceType, data_transfer.ServiceType}
```

Comment:
Let's not start it automatically now. We need to prepare other components for it. 
Later we will enable it for sure.